### DISCUSSION TITLE
Home Assistant OS VM / long time no change after fresh installation

### DISCUSSION THREAD

**Discussion by lebinho** [tteck/Proxmox#3842] [Help Needed]
I decided to restart my Home assistant yesterday (everything was working fine) after mqqt2zigbee instalation and yesterday encountered this line running for ages "A start job is running for HAOS swap" in console while starting. I tried to fix it but nothing so decided to reinstall from scratch using this script "Home Assistant OS VM" and it just completed but... again I see "A start job is running for HAOS swap". The VM was given 12GB RAM, 32 GB disk space, 2 cpu... so it is not resource problem.  Leaning towards thinking that it has somethig to do with proxmox swap but I did run this and looks fine...
NAME      TYPE      SIZE USED PRIO
/dev/dm-0 partition   8G 256K   -2

Troubleshooting actions I did that did not work:
- reinstalling VM and HAOS
- clearing and increasing swap from 8 do 12 gb

Help please

**Comment by Mellowlynx** [comment]
I just updated and rebooted my HA VM, and mine starts just fine.
OS 13.1 - HA Core: 2024.10.1

I did some googling, and found that it might take 20+ minutes to complete before finally continuing booting.
There might be some steps you might need to do or try to fix. Maybe someone here has experienced this too and can assist you.

**Comment by Mellowlynx** [comment]
Also, what is shown in the "Summery" tab of the VM in Proxmox?
Is the usage far too low, or is the system busy and is the usage changing? If so, let it run and see what happens.

**Reply by lebinho** [reply]
Everything is very low. I did try to install 10 times now with no success. Each time the same swap issue. Tried to install with image mounting but it fails on Supervisor issue like here:
https://community.home-assistant.io/t/error-returned-from-supervisor-system-is-not-ready-with-state-setup/413084/4


just found that forum so will re-install and do that ha > banner to see if it works for me as it did for others.
